crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -55,28 +55,32 @@ def cancel
 
       protected
 
+      def crowbar
+        Api::Crowbar.new
```

Comment:
But this gets removed again in the next commit which turns `Api::Crowbar` methods into class methods, so why is it being added?  I guess the answer is "because it's a backport of #674 which already did it that way"?  If so, OK, but please try to avoid that kind of thing in future, since it's pretty confusing to review a PR which makes some changes and then fully or partially reverts them in a later commit in the same PR.

Personally I usually end up doing quite a lot of fixups/squashing on my private branches before submitting PRs, to try to avoid this kind of thing.  Yes, it's more work, but a lot of it can be automated with the right git tools, and keeping the history really clean helps reviews and backports, and makes it easier for anyone reading the history later on to understand the changes.

Hope that makes sense :-)